### PR TITLE
Pricing UI improvements

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Fixed layout spacing and only show checkmark if company is Education.

## Motivation and Context
Checkout Flow epic

## How Has This Been Tested?
Locally and on [stage-18]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc some things I noticed in the UI:
- Buttons (Send Quote and Subscribe) should be equal length
- Space between 5 & Display licenses should be wider
- Space between x & checkmark and price should be wider
- Vertical spacing between 5 Licenses & Per Display & Total should be taller
- What's the behaviour supposed to be for the input element for the number of items?
- What's the plan for Non-profit discount, current pricing page says "$55 per year Education and Non-Profit Discount" but we don't currently show that as a discount for the new page, and the current Education wording does not apply.

Will review code separately. Thanks.

File: web/partials/purchase/checkout-plan-picker.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
 <!-- Step 0 -->
 <div class="checkout-plan-picker">
 
-  <div class="row">
-    <div class="col-md-4 mb-5 madero-style">
+  <div class="stack-md-sideways-panels">
+    <div class="brandscript-panel madero-style mb-5">
```
Comment (alex-deaconu) [inline web/partials/purchase/checkout-plan-picker.html:5]: @ezequielc is the `madero-style` needed here? Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good, just a few small nitpicking points on the UI. Thanks for the fixes!

File: web/partials/purchase/checkout-plan-picker.html (hunk 1 above)
Comment (ezequielc) [reply web/partials/purchase/checkout-plan-picker.html:5]: @alex-deaconu I will double check and remove in a future PR if not needed. Thanks!

**Comment by ezequielc** [comment]
Thanks @alex-deaconu I added the list of UI improvements as low priority in our TODO list.

**Comment by alex-deaconu** [comment]
@ezequielc sounds good, thanks!

**Pull request merged**